integrationm/scripts/_audit.groovy

```diff
@@ -1,77 +1,27 @@
 import groovy.transform.Field
-import org.codehaus.jettison.json.JSONObject
-
 import com.google.common.cache.*
-import java.util.concurrent.TimeUnit
+import com.cultofbits.customizations.audit.AuditUtils
 
-if (msg.product != "recordm-definition" && msg.product != "recordm" ) return
+if (msg.product != "recordm-definition" && msg.product != "recordm") return
```

Comment:
This will never be true `msg.product != "recordm-definition"`. You can remove this line.